blog/releases/announcing-knative-v0-23-release.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Version 0.23 release"
+linkTitle: "Version 0.23 release"
```

Comment:
see above and https://docs.google.com/spreadsheets/d/1p1_kBUd6ZvonxHkMcEJPayf6QIpExuFf5cFq0ptar7I/edit#gid=0